plugin.video.nhklive/resources/lib/scraper.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# KodiAddon (NHK TV Live)
+#
+from t1mlib import t1mAddon
+import re
+import xbmcplugin
+import xbmc
+import calendar
```

Comment:
used?

Reply:
I don't think so, at least I don't see any usages

Reply:
Missed that one, Thanks. Done